index.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <style>
+        body {
+          font-family: "Lato", sans-serif;
+        }
+        
+        .sidenav {
+          height: 100%;
+          width: 0;
+          position: fixed;
+          z-index: 1;
+          top: 0;
+          left: 0;
+          background-color: #111;
+          overflow-x: hidden;
+          transition: 0.5s;
+          padding-top: 60px;
+        }
+        
+        .sidenav a {
+          padding: 8px 8px 8px 32px;
+          text-decoration: none;
+          font-size: 25px;
+          color: #818181;
+          display: block;
+          transition: 0.3s;
+        }
+        
+        .sidenav a:hover {
+          color: #f1f1f1;
+        }
+        
+        .sidenav .closebtn {
+          position: absolute;
+          top: 0;
+          right: 25px;
+          font-size: 36px;
+          margin-left: 50px;
+        }
+        
+        @media screen and (max-height: 450px) {
+          .sidenav {padding-top: 15px;}
+          .sidenav a {font-size: 18px;}
+        }
+        #navbar { position: fixed; }
+        </style>
+        </head>
+        <body>
+        
+        <div id="mySidenav" class="sidenav">
+          <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+          <a href="#Abstract">Abstract</a>
+          <a href="#Introduction">Introduction</a>
+          <a href="#JavaScript_and_Java">Javascript and Java</a>
+          <a href="#comments">Comments</a>
+          <a href="#Notes">Notes</a>
+        </div>
+        
+     
+        <span style="font-size:30px;cursor:pointer" onclick="openNav()">&#9776; open</span>
+        
+        <script>
+        function openNav() {
+          document.getElementById("mySidenav").style.width = "250px";
+        }
+        
+        function closeNav() {
+          document.getElementById("mySidenav").style.width = "0";
+        }
+        </script>
+
+</head>
+<style>
+@media (max-width: 600px)
+{
+  .facet_sidebar
+   {
+    display: none;
+   }
+}
+</style>
+<body>
+    <main id ="main-doc">
+
+        <nav id="navbar" >
+            <header>Navigate through the links:</header>
+
+            <a class="nav-link" href="#Abstract">Abstract</a> <br>
+            
+            <a class="nav-link" href="#Introduction">Introduction</a><br>
+            
+            <a class="nav-link" href="#comments">comments</a><br>
+            
+            <a class="nav-link" href="#JavaScript_and_Java">JavaScript and Java</a><br>
+            
+            <a class="nav-link" href="#Notes">Notes</a><br>
```

Comment:
### Identation
Try to be careful about blank spaces between your HTML lines since it is not useful and makes your code seems bigger

Reply:
- [ ] It will be good to modify this comment and make your request as a suggestion way as I mentioned before.